File: Year 1/Semester 2/Java/Lessons/input.java
import java.util.Scanner;

public class input {

    public static void main(String[] args) {
        Scanner entry = new Scanner(System.in);
        String scanned = entry.next();
        // next input
        
        // conversion
        int x = Integer.parseInt(scanned);

        System.out.println("\n" + scanned);

    }
}

File: Year 1/Semester 1/C/Lessons/input.c
#include <stdio.h>

int main() {
    char scanned[100];  // Buffer to hold the input string

    // Taking input from the user
    printf("Enter a number: ");
    scanf("%s", scanned);  // Read a string input

    // Conversion to integer
    int x = atoi(scanned);  // Convert string to integer

    // Print the scanned input
    printf("\n%s\n", scanned);
    // Optionally, print the converted integer
    printf("Converted integer: %d\n", x);

    return 0;
}
